### PR TITLE
Add ScotRail

### PR DESCRIPTION
**Pull request by ohheyrj**
Scotrail is the main rail operator in Scotland, UK. Their site allows you to book tickets for trains within Scotland for pickup at stations or to be loaded onto a smartcard. Their site does not support 2FA.

### CODE REVIEW

File: _data/transport.yml [hunk 1]
```diff
@@ -165,6 +165,13 @@ websites:
       img: sas.png
       tfa: No
 
+    - name: Scotrail
```
Comment (jamcat22) [inline _data/transport.yml]: Their name appears to be stylized as "ScotRail" according to their website and social media profiles.
Please update their name to be `ScotRail`.

File: _data/transport.yml [hunk 2]
```diff
@@ -165,6 +165,13 @@ websites:
...
+      url: https://www.scotrail.co.uk
+      twitter: Scotrail
```
Comment (jamcat22) [inline _data/transport.yml]: Their Twitter username is stylized as `ScotRail`.
Please update this line to reflect that.

File: _data/transport.yml [hunk 3]
```diff
@@ -165,6 +165,13 @@ websites:
...
+      facebook: Scotrail
```
Comment (jamcat22) [inline _data/transport.yml]: Their Facebook username is stylized as `ScotRail`.
Please update this line to reflect that.

**Review by jamcat22 — CHANGES_REQUESTED** [review]

**Comment by ohheyrj** [comment]
Hello, I have made the above branding changes as requested.

**Comment by Carlgo11** [comment]
So uhm, what's the worst thing that could happen if someone got hold of a ScotRail account? Could they make purchases from that account?

**Comment by ohheyrj** [comment]
Yes they could. You can also purchase season tickets and additional smartcards to hold etickets.

**Review by jamcat22 — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]

**Pull request merged**